dsl/src/main/scala/com.crobox.clickhouse/dsl/FromQuery.scala

```diff
@@ -2,8 +2,14 @@ package com.crobox.clickhouse.dsl
 
 sealed trait FromQuery extends Query with OperationalQuery {
   override val internalQuery = InternalQuery(from = Some(this))
+  val alias: Option[String]
+  val `final`:Boolean
```

Comment:
meh i really don't like the backticked variables, i know why it's there. But maybe we can name it  `useFinal`?

Reply:
Ok Ok ok, changed it to 'finalized' (without quotes)